compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -77,9 +77,28 @@ static llvm::cl::opt<bool> clLLVMGPUEnablePrefetch(
 
 llvm::raw_ostream &operator<<(llvm::raw_ostream &os,
                               const LLVMGPUPipelineOptions &options) {
+  if (options.reorderStrategy) {
+    StringRef reorderStr;
+    if (options.reorderStrategy.value() == ReorderWorkgrupsStrategy::Transpose)
```

Comment:
nit: you can compare directly with `==`
```suggestion
    if (options.reorderStrategy == ReorderWorkgrupsStrategy::Transpose)
```